pallets/inflation/src/lib.rs

```diff
@@ -328,27 +345,46 @@ pub mod pallet {
 
             Ok(().into())
         }
+
+        /// Used to force-set the decay factor for reward payouts.
+        ///
+        /// Must be called by `root` origin.
+        #[pallet::call_index(3)]
+        #[pallet::weight(T::WeightInfo::force_set_decay_factor())]
+        pub fn force_set_decay_factor(
```

Comment:
Shouldn't this be used to set the `decay_rate` on the config though?

I assume the order of actions will be:
* runtime upgrade, with decay rate set to 1.00, and decay factor set to 1.00
* governance action to force set the decay rate to e.g. `0.999....9`

Reply:
I see, makes sense.

What do we need this `force_set_decay_factor` for though?

Reply:
Ok, it has been removed